test/test_cuda.py

```diff
@@ -200,7 +200,9 @@ def tmp(t):
     ('mode', small_3d, lambda t: [],),
     ('mode', small_3d, lambda t: [1], 'dim'),
     ('remainder', small_3d, lambda t: [3], 'value'),
+    ('remainder', small_3d, lambda t: [-3], 'negative_value'),
     ('remainder', small_3d, lambda t: [small_3d_positive(t)], 'tensor'),
+    ('remainder', small_3d, lambda t: [0 - small_3d_positive(t)], 'negative_tensor'),
```

Comment:
To test the case where the dividor is a tensor that can contains negative values, I wanted to pass it `-small_3d_positive` but received an error `AttributeError: 'LongTensor' object has no attribute 'neg'`

Is it a problem with my install / a design decision / just a missing function that I should implement?